Newsweek Project completed

xtrmdarc opened the pull request:
### I have completed the Newsweek project, which I developed on my own using Bootstrap 4 as suggested and completely adapted the site to Laptop, Tablet, and Mobile dimensions.

- [x] Added Navbar
- [x] Added main section articles
- [x] Added in the magazine articles
- [x] Added conversations articles
- [x] Added editors pick articles
- [x] Added slideshows article
- [x] Added all topics articles
- [x] Added footer
- [x] Worked on responsiveness
- [x] Complete readme
- [x] Added favicon

xtrmdarc committed:
header and navbar added and styled

xtrmdarc committed:
main basic layout created

xtrmdarc committed:
complete features stories and top story, added aside and styled

xtrmdarc committed:
culture and travel section added and styled

xtrmdarc committed:
complete main story column

xtrmdarc committed:
editors pick added and completed

xtrmdarc committed:
in the magazine section added and styled and newsweek conversations added and styled

xtrmdarc committed:
featured slideshows added and styled

xtrmdarc committed:
articles sections added and styled

xtrmdarc committed:
subscribe and newsletter cta sections added and styled

xtrmdarc committed:
footer added and styled

xtrmdarc committed:
responsive footer and promo sections

xtrmdarc committed:
handling positioing mobile responsive

xtrmdarc committed:
complete responsiveness

xtrmdarc committed:
format readme

xtrmdarc committed:
fix html validator

xtrmdarc committed:
fix stickler

xtrmdarc committed:
add favicon and fix readme

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+button {
+  border: none;
+  font-weight: bold;
```
Replace with bootstrap classes

xtrmdarc replied:
Adding `btn` bootstrap class to  all button elements

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+
+a:hover {
+  color: inherit;
+  text-decoration: none;
```
Replace with bootstrap class

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+}
+
+main {
+  padding: 10px 2.5% !important;
```
Don't use !important

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+.hamburguer_menu {
+  background-color: transparent;
+  color: rgb(230, 230, 230);
+  display: none;
```
Use bootstrap class

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+
+.buttons_signin {
+  display: flex;
+  justify-content: flex-end;
```
Use bootstrap classes

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+
+.black_red_link:hover {
+  color: red;
+  text-decoration: none;
```
Use bootstrap class

xtrmdarc replied:
Hi! believe that changing the color text to red and removing the underline decoration from an anchor are project-specific functionalities, therefore, I can't use bootstrap classes for this .

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+
+footer .navigation ul {
+  padding-left: 0;
+  margin-left: 0;
```
Use bootstrap classes

mosesogwo commented on assets/styles.css:
```diff
@@ -0,0 +1,930 @@
+.social_icons_set {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
```
Use bootstrap classes

mosesogwo reviewed (changes requested):
### Hi @xtrmdarc, you have done well:+1:. We need to make some changes for this to get approved:

1. Let us adjust our responsive break points to fit the following: 
**< 768px - small** screen, 
**768px - 1000px - medium** screen 
and **> 1000px - large** screen.

2. Our Subscribe and Sign in links get distorted from around 470px screen size. Let us fix this
![image](https://user-images.githubusercontent.com/46611870/73785620-ebcdfa80-4797-11ea-95e5-95b1deda67d5.png)

3. It is bad css practice to use `!important`. Let us do away with them.

4. Let us use bootstrap classes wherever possible. See comments on your stylesheet. 

_**Kindly make these changes and submit another code review request,**_

**Good luck:smile:**

### Status: Changes requested.

xtrmdarc committed:
fix review issueues

xtrmdarc commented:
Thanks for your code review! It was really helpful. I wanted to note that I am building project number seventh of the curriculum, therefore, I am using the suggested bootstrap breakpoints for my project. Therefore, I will use >992 for Laptop,  <992 & >768 for Tablet and <768 for mobile. I can't really switch to other breakpoints at this moment because it will move all my design since I am matching bootstrap breakpoints for col, col-sm, col-md, and col-lg. Please, kindly see the attached image for suggested bootstrap breakpoints. Also, I have removed most of the `!important` keywords in my CSS, I'm just leaving the necessary ones to make my navbar responsive and sticky through all dimensions.
![image](https://user-images.githubusercontent.com/12993913/73791280-01c7c500-4770-11ea-9002-6e62015bb823.png)

xtrmdarc committed:
suggested bootstrap breakpoints added

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.hamburguer_menu {
+  display: none;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  font-size: 12px;
+  color: gray;
+  font-weight: bold;
+  display: inline-block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.featured_topic {
+  color: rgb(156, 156, 156);
+  font-size: 14px;
+  display: block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.l_b_topic_tag {
+  position: absolute;
+  display: inline-block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.nl_logo {
+  display: inline-block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  font-size: 13px;
+  color: rgb(44, 44, 44);
+  font-weight: 800;
+  display: block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.culture_travel .info_wrapper .tag {
+  display: inline-block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.section_title {
+  display: flex;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.see_all_link {
+  display: block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  width: 50px;
+  height: 50px;
+  opacity: 1;
+  display: flex;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.social_icons_set span {
+  display: inline-block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+@media screen and (min-width: 768px) and (max-width: 992px) {
+  .main_nav {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .right_half_wing {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .header_logo .text_date {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  #right_half_wing {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .hamburguer_menu {
+    display: block !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+@media screen and (max-width: 767px) {
+  .main_nav {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .right_half_wing {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .header_logo .text_date {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  #right_half_wing {
+    display: none !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .hamburguer_menu {
+    display: block !important;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  footer .newsweek_logo_red span {
+    position: static;
+    margin: 0;
+    display: block;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .position_handler {
+    display: flex;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .go_btn {
+    display: none;
```
Use a [bootstrap class for display](https://getbootstrap.com/docs/4.4/utilities/display/) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+  .position_handler {
+    display: flex;
+    flex-direction: column;
```
Use a [bootstrap class for flex-direction](https://getbootstrap.com/docs/4.4/utilities/flex/#direction) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .sub_news .content-cta {
+    flex-direction: column;
```
Use a bootstrap class for [flex-direction](https://getbootstrap.com/docs/4.4/utilities/flex/#direction) for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+  .sub_news .content-cta {
+    flex-direction: column;
+    justify-content: center;
```
Use a bootstrap class for [justify-content ](https://getbootstrap.com/docs/4.4/utilities/flex/#justify-content)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
```
Use a bootstrap class for [justify-content ](https://getbootstrap.com/docs/4.4/utilities/flex/#justify-content)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  height: 50px;
+  opacity: 1;
+  display: flex;
+  justify-content: center;
```
Use a bootstrap class for [justify-content ](https://getbootstrap.com/docs/4.4/utilities/flex/#justify-content)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.social_icons_set .row {
+  justify-content: flex-end;
```
Use a bootstrap class for [justify-content ](https://getbootstrap.com/docs/4.4/utilities/flex/#justify-content)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  }
+
+  .social_icons_set .row {
+    justify-content: flex-start;
```
Use a bootstrap class for [justify-content ](https://getbootstrap.com/docs/4.4/utilities/flex/#justify-content)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.section_title {
+  display: flex;
+  align-items: center;
```
Use a bootstrap class for [align-items](https://getbootstrap.com/docs/4.4/utilities/flex/#align-items)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  opacity: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Use a bootstrap class for [align-items](https://getbootstrap.com/docs/4.4/utilities/flex/#align-items)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  background-color: #f4f5f6;
+  padding: 40px;
+  margin-top: 10px;
+  align-items: center;
```
Use a bootstrap class for [align-items](https://getbootstrap.com/docs/4.4/utilities/flex/#align-items)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.align-flex-start {
+  align-items: flex-start;
```
Use a bootstrap class for [align-items](https://getbootstrap.com/docs/4.4/utilities/flex/#align-items)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.nl_header_wrapper {
+  align-items: center;
```
Use a bootstrap class for [align-items](https://getbootstrap.com/docs/4.4/utilities/flex/#align-items)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.section_title {
+  display: flex;
+  align-items: center;
+  text-align: center;
```
Use a bootstrap class for [text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.see_all_link {
+  display: block;
+  padding: 8px 0;
+  text-align: center;
```
Use a bootstrap class for [text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  .sub_news .content-cta {
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
```
Use a bootstrap class for [text-align](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment)for the according screen size directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.gray_white_block {
+  background: gray;
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.newsweek_logo_red {
+  color: white;
```
Use the bootstrap class .text-white directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.red_white_block {
+  background: red;
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.text_date {
+  font-size: 14px;
+  font-weight: 100;
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.conversations .main_wrapper {
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.newsweek_logo {
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.conversations .red_underline_link {
+  color: white;
```
Use the bootstrap class **.text-white** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.black_red_link {
+  color: black;
```
Use the bootstrap class **.text-dark** or **text-body** directly in your HTML file instead

xtrmdarc replied:
Can't remove add text-dark to the elements because it will override my hover selector too.

xtrmdarc replied:
Can't remove add text-dark to the elements because it will override my hover selector too.

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+header .main_nav ul li a {
+  font-weight: bold;
+  color: black;
```
Use the bootstrap class **.text-dark** or **text-body** directly in your HTML file instead

xtrmdarc replied:
Can't remove add text-dark to the elements because it will override my hover selector too.

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.red_underline_link {
+  color: black;
```
Use the bootstrap class **.text-dark** or **text-body** directly in your HTML file instead

xtrmdarc replied:
Can't remove add text-dark to the elements because it will override my hover selector too.

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.opinion_articles article .opinion_info span {
+  font-size: 12px;
+  color: gray;
```
Use the bootstrap class **.text-muted** or **text-black-50** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  display: inline-block;
+  background: rgb(184, 214, 240);
+  padding: 3px;
+  color: gray;
```
Use the bootstrap class **.text-muted** or **text-black-50** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+footer .newsweek_logo_red span {
+  position: absolute;
+  bottom: -3px;
+  color: gray;
```
Use the bootstrap class **.text-muted** or **text-black-50** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  margin-top: 15px;
+  padding-top: 10px;
+  font-size: 13px;
+  color: gray;
```
Use the bootstrap class **.text-muted** or **text-black-50** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.text_date {
+  font-size: 14px;
+  font-weight: 100;
```
Use the bootstrap class **.font-weight-light** or **font-weight-lighter** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+header .main_nav ul li a {
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+header .main_nav ul li:last-child a {
+  font-size: 15px;
+  font-weight: 300;
```
Use the bootstrap class **.font-weight-normalt** or **font-weight-light** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.featured_article h3 {
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.opinion_articles article .opinion_info h4 {
+  font-size: 15px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.opinion_articles article .opinion_info span {
+  font-size: 12px;
+  color: gray;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  display: block;
+  text-transform: uppercase;
+  margin-bottom: 7px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  left: 0;
+  padding: 9px;
+  font-size: 15px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.latest_news h4 {
+  font-size: 15px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.signup_newsletter h4 {
+  font-size: 16px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.signup_newsletter .cta {
+  font-size: 13px;
+  color: rgb(44, 44, 44);
+  font-weight: 800;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.culture_travel .info_wrapper h3 {
+  font-size: 20px;
+  font-weight: 300;
```
Use the bootstrap class **.font-weight-light** or **font-weight-normal** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  padding: 3px;
+  color: gray;
+  font-size: 12px;
+  font-weight: 300;
```
Use the bootstrap class **.font-weight-light** or **font-weight-lighter** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.more_stories article h4 {
+  font-size: 14px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.more_stories article p {
+  font-size: 14px;
+  font-weight: 300;
```
Use the bootstrap class **.font-weight-light** or **font-weight-normal** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.section_title h2 {
+  margin: 0 8px;
+  font-size: 18px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+  display: block;
+  padding: 8px 0;
+  text-align: center;
+  font-weight: 800;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.in_the_mag .articles_wrapper .tags span {
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.in_the_mag .articles_wrapper h3 {
+  font-size: 18px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.more_conversations h4 {
+  font-size: 15px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.featured_slideshows .articles_wrapper h3 {
+  font-size: 20px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+.topic_section .topic_text {
+  font-size: 20px;
+  color: red;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.topic_section .articles_wrapper h3 {
+  font-size: 18px;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.topic_section .secondary_articles h4 {
+  font-size: 14px;
+  font-weight: 800;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.sub_news .content-cta p {
+  font-weight: 100;
```
Use the bootstrap class **.font-weight-light** or **font-weight-lighter** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+
+.sub_news .promo_text {
+  color: red;
+  font-weight: bold;
```
Use the bootstrap class **.font-weight-bold** or **font-weight-bolder** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+header .main_nav ul li:last-child {
+  border: none;
```
Use the bootstrap class **.border-0** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.vertical_articles_bot_bordered li:last-child {
+  border: none;
```
Use the bootstrap class **.border-0** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.border-left-nav li:first-child {
+  border: none;
```
Use the bootstrap class **.border-0** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.language-nav li:nth-child(2) {
+  border: none;
```
Use the bootstrap class **.border-0** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.site-nav li:last-child {
+  border: none;
```
Use the bootstrap class **.border-0** directly in your HTML file instead

euqueme commented on assets/styles.css:
```diff
@@ -0,0 +1,927 @@
+}
+
+.buttons_signin button {
+  border-radius: 0;
```
Use the bootstrap **.rounded-0** directly in your HTML file instead

euqueme reviewed (changes requested):
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @xtrmdarc :handshake: 

### First of all :clap: :clap: :clap: claps to you!! this project is amazing and looks just like the original I am truly blown away :astonished: you've even included the script to make the navbar small when you scroll down the page amazing!! :star2: 

### All you have to do now is use a little more bootstrap within your HTML, The CSS has to be as small as it can be since this project is all about using bootstrap:

- [ ] I see you used the bootstrap responsive breakpoints so all the required changes in your code are for all bootstrap classes that have breakpoint responsiveness and also aren't used in hovers or are within classes that aren't affected by a media query. 

[OPTIONAL] I strongly recommend using bootstrap classes for margin, padding, borders, colors, background colors if possible.

that is it! :raised_hands:  it has been a blast to review this project :guitar: 

### Please :pray:  do the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

xtrmdarc commented:
@euqueme Thank you so much for your insight, I will work on that :)

xtrmdarc committed:
using bootstrap utiliies tools

xtrmdarc committed:
fix small text align bug

xtrmdarc commented:
Hi @euqueme Thanks for your code review. I just fixed all the issues with the project. Added all the utility shorthands provided by bootstrap. However, I cannot use text-dark in some h# tags because it will override my hover selector with an `!important` keyword.

nikasulo reviewed (changes requested):
# Great job @xtrmdarc ! :rocket: 
Your project is really solid and the amount of work you've put into this is glaring! I understand why you would think you need to use the `!important` tag in your CSS, however, being that your markup relies solely on classes to select HTML nodes, it's only a matter of understanding specificity in how you select your elements. As such, please reconsider your approach and remove these tags because, in addition to the fact that they're largely a bad practice, they are really not necessary on this occasion.

Please consider making this change and submit for your next review! I wish you the best :100: See you at the very top! :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

xtrmdarc committed:
remove important keywords from css, new approach

xtrmdarc commented:
Hi @Oluwadamilareolusakin, Thanks for your review. I reconsidered the approach of the `!important `tags being used and I removed. Thanks for your explanation

nikasulo commented:
@xtrmdarc way to go! I apologize for turning in your review so late. I wrote it almost 6 hours ago, however, my internet went out and only returned about an hour ago. This is not standard practice at Microverse and as such, I apologize for any inconvenience or delay I caused you, I am sorry. Please let me know if you have any questions or need advice on anything, whether it's a project in the curriculum, a concept or you just want to chat. I would be happy to help :smile: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

euqueme reviewed (approved):
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE) :woman_technologist: 

### Hello @xtrmdarc  :raised_hands: 

#### Thank you for doing all the changes, this project is hands down :no_good_woman:  the best I've reviewed so far Congratulations!! :fireworks:  and for many more projects like this :clinking_glasses: 

PROJECT STATUS: APPROVED :green_apple:

xtrmdarc committed:
Update README.md
